### ISSUE TITLE
Create a more Advance UI

### ISSUE THREAD

**Issue by rpekash** [rpekash/Challenge-1#9]
Currently using a console based UI, implementing a more Advance UI that Pops up after running the program would help out with user experience and functionality.

**Comment by Sharktroid** [comment]
Yes, especially as it would mean that the user could use a mouse instead of typing everything.
